tests/Fixtures/Autoloaded/ProtectedVisibility/pv-final.php

```diff
@@ -4,9 +4,9 @@
 
 class Test
 {
-    protected final function foo(int $param, $test = 1) : bool
```

Comment:
This way we test if we don't mutate whether a method passes by reference. As this could cause confusion in the diff.